server/model/dbSetupQuery.js

```diff
@@ -0,0 +1,40 @@
+const users = `
+DROP TABLE IF EXISTS users cascade;
+CREATE TABLE users(
+  userId SERIAL PRIMARY KEY,
+  fullName VARCHAR(60) NOT NULL,
+  email VARCHAR(40) NOT NULL 
+);`;
+
+const authentication = `
+DROP TABLE IF EXISTS authentication cascade;
+CREATE TABLE authentication(
+  userId INT,
+  username VARCHAR(30) NOT NULL UNIQUE,
+  password VARCHAR(40) NOT NULL,
```

Comment:
you may want a much longer field size. Remember the password would ideally be hashed then stored in the dB.
use a much longer field size or maybe just use text.